README.md

```diff
@@ -31,7 +31,8 @@ the Go programming language:
 Working with the `gunk` command line tool should be instantly recognizable to
 experienced Go developers:
 
-	gunk generate ./examples/util
+    cd ./examples/util
+	gunk generate .
```

Comment:
don't mix tabs and spaces, and also `gunk generate` can be used instead of `gunk generate .`

Reply:
Opps, I'll fix that!

Reply:
I tried running with `gunk generate` and I get the error `error: no Gunk packages to generate`.

Reply:
Huh... please leave it as it is then, and file an issue :)

Reply:
Done!